lib/matplotlib/backends/backend_wx.py

```diff
@@ -1253,29 +1252,27 @@ class FigureManagerWx(FigureManagerBase):
 
     Attributes
     ----------
-    canvas : `FigureCanvas`
-        a FigureCanvasWx(wx.Panel) instance
+    canvas : FigureCanvasWx
     window : wxFrame
-        a wxFrame instance - wxpython.org/Phoenix/docs/html/Frame.html
```

Comment:
The link is broken and if anything the only link should be to the wxpython general docs at the top of the module (perhaps).